marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -476,7 +476,9 @@ private void writeResourceFile(String srcFile, Path dstFile) {
 
     private void writeResourceFile(String srcFile, Path dstFile, boolean overwrite) {
         if (overwrite || !dstFile.toFile().exists()) {
-            logger.info("Getting file: " + srcFile);
+            if (logger.isDebugEnabled()) {
```

Comment:
I changed this to debug, as it was getting logged a ton when running a test, but not really adding any value.